README.md

```diff
@@ -9,7 +9,7 @@ See the [cisco.nd collection index](https://galaxy.ansible.com/ui/repo/published
 
 ## Requirements
 
-- Ansible v2.16 or newer
```

Comment:
We should probably also update the runtime.yml. Furthermore I think we should create the same PRs to aci and mso collection.